hazelcast/src/main/java/com/hazelcast/internal/diagnostics/MetricsPlugin.java

```diff
@@ -87,23 +88,8 @@ public void run(DiagnosticsLogWriter writer) {
         private long timeMillis;
 
         @Override
-        public void renderLong(String name, long value) {
-            writer.writeSectionKeyValue(SECTION_NAME, timeMillis, name, value);
-        }
-
-        @Override
-        public void renderDouble(String name, double value) {
-            writer.writeSectionKeyValue(SECTION_NAME, timeMillis, name, value);
-        }
-
-        @Override
-        public void renderException(String name, Exception e) {
-            writer.writeSectionKeyValue(SECTION_NAME, timeMillis, name, e.getClass().getName() + ':' + e.getMessage());
-        }
-
-        @Override
-        public void renderNoValue(String name) {
-            writer.writeSectionKeyValue(SECTION_NAME, timeMillis, name, "NA");
+        public void render(CharSequence key, long value) {
+            writer.writeSectionKeyValue(SECTION_NAME, timeMillis, key.toString(), value);
```

Comment:
This can generate a lot of litter if the key isn't a String.

Reply:
Do I have permission to change `DiagnosticsLogWriter` interface? Otherwise there is little we can do about it.